### PR TITLE
chore(ci): Create issue on sitespeed failure MAASENG-2176

### PR DESCRIPTION
**Pull request by ndv99**
## Done

- Added failure condition to sitespeed.io workflow
- Created universal CI failure issue template
- Directed all non-PR workflows to use template

## Fixes

Fixes [MAASENG-2176](https://warthogs.atlassian.net/browse/MAASENG-2176)

[MAASENG-2176]: https://warthogs.atlassian.net/browse/MAASENG-2176?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .github/workflows/upload.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: Upload
+name: Build upload
```
Comment (petermakowski) [inline .github/workflows/upload.yml:1]: Love these drive-by improvements, well done.

**Comment by ndv99** [comment]
@petermakowski https://github.com/ndv99/maas-ui/issues/3 Run failed (on purpose), and the issue was created successfully!

**Review by petermakowski — APPROVED** [review]
